internal/compiler-bridge/src/main/scala/xsbt/Dependency.scala

```diff
@@ -110,6 +110,7 @@ final class Dependency(val global: CallbackGlobal) extends LocateClassFile with
     val memberRef = processDependency(DependencyByMemberRef, false) _
     val inheritance = processDependency(DependencyByInheritance, true) _
     val localInheritance = processDependency(LocalDependencyByInheritance, true) _
+    val macroExpansion = processDependency(DependencyByMacroExpansion, false) _
```

Comment:
I would sugest naming it `scala2MacroExpansion` to avoid any confusion if we need something similar in the future for Scala 3 macros.